src/data/faq.json

```diff
@@ -101,6 +101,170 @@
+
+                        ]
+                    },
+                    {
+                        "title": "Can you also store in the digital vaccination certificate that you have already been infected or tested negative?",
+                        "anchor": "vaccination_certificate_recovered_negative_test",
+                        "active": true,
+                        "textblock": [
+                            "Yes, negative tests or a recovery from the Corona virus can also be stored in the CovPass-App and Corona-Warn-App as a test certificate or recovery certificate. A recovery from a Corona infection is documented by means of a PCR test result."
+                        ]
+                    },
+                    {
+                        "title": "Why has no joint EU-wide project been called out for proposal for a certificate?",
+                        "anchor": "vaccination_certificate_eu",
+                        "active": true,
+                        "textblock": [
+                            "A joint EU call for proposals would have taken too much time and would have been difficult to implement due to the different vaccination information systems in the Member States. The EU approach is about regulating a recognition framework. Any cross-border pilots could be accompanied by the EU Commission. When implementing the Digital Proof of Vaccination in Germany, the EU requirements are taken into account from the outset."
+                        ]
+                    },
+                    {
+                        "title": "Does vaccination status affect the calculation of the risk? For example, if a vaccinated person has become infected and then warns other users?",
+                        "anchor": "vaccination_certificate_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "No, the risk calculation in the app does not take into account whether you are vaccinated against or recovered from a corona infection. The functions are independent of each other. The quality of your risk assessment does not change depending on whether you choose to save your vaccination certificate in the application, which underlines that this is completely voluntary. However, your personal vaccination status may affect your interpretation of the risk warning."
+
+                        ]
+                    },

```

Comment:
The title "Does vaccination status affect the calculation of the risk? For example, if a vaccinated person has become infected and then warns other users?" is mixing up two different points:

1. If a person has been fully vaccinated then according to RKI the risk of becoming infected is significantly lowered. (Not surprising, since this is the goal of getting vaccinated.) So users could expect the app to take account of this. But at least it is stated that the app doesn't change its calculation.
2. What about if an vaccinated person becomes infected? Well in that case it is irrelevant whether the person has been vaccinated or not. Infected is infected. So hopefully that person will have used exposure window and presence tracing before finding out they were infected and then hopefully they will share their infectious status via CWA, so others can be warned. 

In general this section is an answer to the issue I raised in https://github.com/corona-warn-app/cwa-wishlist/issues/457 "Advice for app use by people who have been vaccinated".

I don't know what "quality of your risk assessment" means. In fact I don't understand the whole sentence "The quality of your risk assessment does not change depending on whether you choose to save your vaccination certificate in the application, which underlines that this is completely voluntary."

The sentence "However, your personal vaccination status may affect your interpretation of the risk warning." does not help the user by offering any sort of practical advice. It is therefore largely useless.

This section would benefit from being reviewed about the message it is trying to bring across.

Reply:
{
                        "titel": "Does my vaccination status affect the calculation of the risk? Does the calculation of the risk change if a vaccinated person has become infected, and then warns other users?",                        
                        "anchor": "vac_cert_risk_assesment",
                        "active": true,
                        "textblock": [
                            "Risk calculation and vaccination certificates are independent features of the app. The risk calculation does not take into account whether you are vaccinated against COVID-19. The calculation of the risk also does not change if a vaccinated person has become infected, and then warns other users."
                        ]
                    },